### PR TITLE
Project standardization

### PR DESCRIPTION
**Pull request by bandophahita**
https://github.com/ScreenPyHQ/screenpy/issues/127
Attempting to make various portions of each screenpy project uniform.

### CODE REVIEW

**Comment by bandophahita** [comment]
Do not review this until https://github.com/ScreenPyHQ/screenpy/issues/127 has been merged.

File: pyproject.toml [hunk 1]
```diff
@@ -15,107 +34,20 @@ extend-exclude = '''
...
-
 
 [tool.ruff]
 target-version = "py38"  # minimum supported version
 line-length = 88  # same as Black.
-
 extend-exclude = [
-    "screenpy_selenium/__init__.py",
-    "screenpy_selenium/__version__.py",
+    "__init__.py",
```
Comment (perrygoy) [inline pyproject.toml]: I probably don't need to comment on this on every one. I missed the one on ScreenPy: PyOTP after all. But same thing here—we probably want to remove this exclusion.

File: screenpy_selenium/__version__.py [hunk 2]
```diff
@@ -1,15 +1,10 @@
 """
- ░█▀▀▀█ █▀▀ █▀▀█ █▀▀ █▀▀ █▀▀▄ ░█▀▀█ █  █   ░█▀▀▀█ █▀▀ █   █▀▀ █▀▀▄  ▀  █  █ █▀▄▀█
-  ▀▀▀▄▄ █   █▄▄▀ █▀▀ █▀▀ █  █ ░█▄▄█ █▄▄█    ▀▀▀▄▄ █▀▀ █   █▀▀ █  █ ▀█▀ █  █ █ ▀ █
- ░█▄▄▄█ ▀▀▀ ▀ ▀▀ ▀▀▀ ▀▀▀ ▀  ▀ ░█    ▄▄▄█   ░█▄▄▄█ ▀▀▀ ▀▀▀ ▀▀▀ ▀  ▀ ▀▀▀  ▀▀▀ ▀   ▀
+     ░█▀▀▀█ █▀▀ █▀▀█ █▀▀ █▀▀ █▀▀▄ ░█▀▀█ █  █   ░█▀▀▀█ █▀▀ █   █▀▀ █▀▀▄  ▀  █  █ █▀▄▀█
+      ▀▀▀▄▄ █   █▄▄▀ █▀▀ █▀▀ █  █ ░█▄▄█ █▄▄█    ▀▀▀▄▄ █▀▀ █   █▀▀ █  █ ▀█▀ █  █ █ ▀ █
+     ░█▄▄▄█ ▀▀▀ ▀ ▀▀ ▀▀▀ ▀▀▀ ▀  ▀ ░█    ▄▄▄█   ░█▄▄▄█ ▀▀▀ ▀▀▀ ▀▀▀ ▀  ▀ ▀▀▀  ▀▀▀ ▀   ▀
```
Comment (perrygoy) [inline screenpy_selenium/__version__.py:4]: 🎆 💮 🎆

**Review by perrygoy — COMMENTED** [review]
Another comment about `__init__.py`!

File: pyproject.toml (hunk 1 above)
Comment (perrygoy) [reply pyproject.toml]: Haha. Maybe that's why i had those files ignored.

We could instead just ignore the specific rules in those files in the `per-file-ignore` section. I don't want to completely turn off `ruff`ing them up, and especially not for *all* of the `__init__.py` files.

Comment (bandophahita) [reply pyproject.toml]: Updated!  I was also able to remove the blanket exclusion for `__version__.py` in each of the projects.

**Review by perrygoy — APPROVED** [review]
Woo! Standards set, standards met.

**Pull request merged**